src/flexmeasures_client/client.py

```diff
@@ -0,0 +1,141 @@
+from __future__ import annotations
+
+import asyncio
+import socket
+from dataclasses import dataclass
+from typing import Any, cast
+
+import async_timeout
+import pandas as pd
+from aiohttp.client import ClientError, ClientSession
+from yarl import URL
+
+
+@dataclass
+class FlexmeasuresClient:
+    """Main class for connecting to the Flexmeasures API"""
+
+    password: str
+    email: str
+    access_token: str = None
+    scheme: str = "http"
+    host: str = "localhost:5000"
+    local: bool = False
+    version: str = "/v3_0/"
```

Comment:
Should say `api_version`, and including the prefixed slash should not be necessary.